docs/guides/serverless-run-first-workload.ipynb

```diff
@@ -0,0 +1,396 @@
+   "metadata": {},
+   "source": [
+    "## Retrieve logs and results\n",
+    "\n",
+    "As mentioned before, once a program is `RUNNING`, you can use `job.logs()` to fetch logs created from `print()` outputs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "cbf12a1b-bfff-4eb2-82f1-298de1e9c114",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "No logs yet.\n"
+     ]
+    }
+   ],
+   "source": [
+    "logs = job.logs()\n",
+    "print(logs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bff87452-bf98-4c7a-85c8-d280d89a7970",
+   "metadata": {},
+   "source": [
+    "Once a program is `DONE`, you can use `job.results()` to fetch the result stored in `save_result()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "bc4fbe20-456c-4ce4-bf26-dfdb8b68e89f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{}\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = job.result()\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "925a8d81-e85b-4e26-a9e1-3a01e8e0785b",
+   "metadata": {},
+   "source": [
+    "At any time, you can also cancel a job:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "c0d999d9-af9e-4312-b5bb-f60f714257ba",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Job has been stopped.'"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "job.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca9791df-60c2-49d8-8541-741078ea63c6",
+   "metadata": {},
+   "source": [
+    "## List previously run jobs run with Serverless\n",
+    "\n",
+    "You can use `get_jobs()` to list all jobs submitted to Qiskit Serverless:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "ba9365c4-b07d-4dc6-9594-edc07c44bc63",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<Job | 0ede4d0f-06d6-4360-86a1-5028ca511b3f>,\n",
+       " <Job | abf78e9a-b554-4e38-966a-f99cff877b8c>,\n",
+       " <Job | 90e1109e-809f-4768-a2dc-f45bf71a97b4>,\n",
+       " <Job | 313050f2-aa78-4d7d-99f4-44bdfe98e4d7>]"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "old_jobs = serverless.get_jobs()\n",
+    "old_jobs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fd3415a0-0e55-4991-8764-c9fe66eb2664",
+   "metadata": {},
+   "source": [
+    "You've successfully run your first Qiskit Serverless program. Next, you can explore [compute and data management tools](./serverless-manage-resources) available to your program, including parallelization."
```

Comment:
This line feels a bit repetitive right before the "next steps" section, maybe just include this link in the next steps section instead

Reply:
done in d62464a90353b1a987f549ad031b66fe80361996